CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Breaking Changes
```

Comment:
@AVHopp, @Scienfitz: Note that I've implement this currently as breaking change since I wanted to hear your thoughts before I add deprecation mechanism. In principle, this is the same change as already done in #289  for the search spaces, and there I introduced a deprecation. I can now either
* do the same here, requiring to replicate the same logic as in the previous PR or
* keep it as a breaking change, but then we must release this under a new version

Let me know

Reply:
One vote with "don't care" and one with "let's do", so I added it. Was also mostly copy-pasting of deprecation code, so no big deal. Removing it in the future will take a few mins only 👍🏼